imageprocessor/templates/index.html

```diff
@@ -1,18 +1,41 @@
 {% extends "base.html" %}
 {% block body %}
+{% load staticfiles %}
 <style>
     h2 { margin: 20px; }
     a { display: block; margin: 20px; }
 </style>
 <div>
-    <h2>Photo Tagger</h2>
-    <a href = '{% url 'classify' %}'>Click here to access Image classifier</a>
-    <a href = '{% url 'tagsearch' %}'>Click here to access tag searcher</a>
-    {% if user.is_authenticated %}
-    <a href = '{% url 'view_my_pictures' %}'>Click here to view your pictures</a>
-    {% else %}
-    <a href = '{% url 'login' %}'>Click here to login</a>
-    <a href = '{% url 'register' %}'>Click here to register</a>
-    {% endif%}
+    <h2>Welcome to Photo Tagger!</h2>
+	<!-- Icons Grid -->
+    <section class="features-icons bg-light text-center">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-3">
+              <a href = '{% url 'classify' %}' style="text-decoration: none !important;"><h3><span><img src="{% static 'img/cloud-upload.png' %}" width="100" height="100" alt=""></span> <br>Image Classifier</h3>
+              <p class="lead mb-0">Upload multiple images to generate tags! <span class="glyphicon glyphicon-search"></span></p></a>
+          </div>
+          <div class="col-lg-3">
+              <a href = '{% url 'tagsearch' %}' style="text-decoration: none !important;"><h3><span><img src="{% static 'img/image-search.png' %}" width="100" height="100" alt=""></span><br>Tag Searcher</h3>
```

Comment:
It's considered bad practice to use !important, it makes writing styles quickly become really unmanageable, and it looks like it's not needed here.

Reply:
Removed it from styling